index.html

```diff
@@ -240,13 +240,35 @@ <h3>
         <h2>
           Powerful features
         </h2>
-        <p>
+        <p data-cite="permissions-policy">
           A <dfn class="export" data-local-lt="feature">powerful feature</dfn> is a web platform
           feature (usually an API) for which a user gives [=express permission=] before the feature
-          can be used. Access to the feature is determined by the user having
-          [=permission/granted=] permission, or by satisfying some criteria that is equivalent to a
-          permission [=permission/grant=].
+          can be used. Except for a few notable exceptions (e.g., the [[[Notifications]]]), most
+          powerful features are also [=policy-controlled features=]. For powerful features that are
+          [=policy-controlled features=], [[[permissions-policy]]] controls whether a [=document=]
+          is [=allowed to use=] a feature, and only if it is allowed can a powerful feature request
```

Comment:
"and only if it is allowed can a powerful feature request" this sounds a bit weird to me, maybe move "can" to the end

Reply:
A couple of possible rewording suggestions:

1. For powerful features that are also policy-controlled features, Permissions Policy controls whether a document is allowed to use a given feature. A powerful feature may only request express permission from a user if the document has permission delegated to it via the corresponding policy-controlled feature.

2. For powerful features that are also policy-controlled features, a powerful feature may only request express permission from a user if and only if a document has permission delegated to it via Permissions Policy.

Reply:
Went with option 1, with a small clarification.